### ISSUE TITLE
How to maximise badge accessibility?

### ISSUE THREAD

**Issue by trexfeathers** [SciTools/.github#216]
Badges - the SVG images we use to communicate repository health - are images of text, which are discouraged by [WCAG 1.4.5 Images of Text](https://www.w3.org/TR/WCAG22/#images-of-text) since plain text is more accessible. But they are the only way to display dynamic content in a GitHub-flavoured-Markdown. We are probably forced to use these, with appropriate alt-text to maximise accessibility.

The more tricky question is what should be done with the static badges (e.g. 'this repository uses ASV' or whatever). We use these since so that they are consistent with the dynamic badges, making for a more visually readable list of repository health markers. But for 'non-visual' reading (screen readers) they are LESS readable. We should make an explicit decision on these, possibly based on testing with a screen reader:

- Replace with plain text?
- Keep as a badge, with the best alt-text we can manage?
- Discourage users of screen readers from attempting to read the health section?

Again, the most important thing is to consider this, even if measured decision is to leave it.
